backend/trafficrules/src/main/resources/db/migration/V1__Add_table_users.sql

```diff
@@ -0,0 +1,7 @@
+create table users
```

Comment:
A small neat: let's have the sql keywords as uppercase everywhere for consistency

Reply:
Changed the sql keywords in uppercase